pydis_site/apps/resources/resources/software_design_by_example.yaml

```diff
@@ -0,0 +1,21 @@
+name: Software Design by Example
+description: A tool-based introduction to Software Design with Python. This book teaches
+  design by explaining examples of small versions of familiar tools to show how
+  experienced software designers think. It introduces some fundamental ideas in computer
+  science that many self-taught programmers haven’t encountered.
+title_image: https://third-bit.com/sdxpy/logo.svg
+title_url: https://third-bit.com/sdxpy/
+urls:
+- icon: branding/github
+  url: https://github.com/gvwilson/sdxpy
+  color: black
```

Comment:
Similar to Fluent Python and the other books, we could also include a link to the goodreads page here: https://www.goodreads.com/book/show/199430059-software-design-by-example

Reply:
Thank you for reviewing this!

Nice addition, thanks for the link!